app/views/question/_semantic_recognition.html.erb

```diff
@@ -0,0 +1,12 @@
+<p>Please select the best category that describes the scene in the picture:</p>
+
+<div id="radio_buttons">
+  <%= radio_button_tag :semantic, 'indoor', :checked => {params[:semantic] => 'indoor'} %>
```

Comment:
I couldn't figure out how to do this. I've tried multiple things, but whatever I do, the answer is never stored in the params map.